example/src/main/scala/HelloWorldAdvanced.scala

```diff
@@ -20,9 +20,9 @@ object HelloWorldAdvanced extends App {
   }
 
   private val server =
-    Server.port(PORT) ++              // Setup port
-      Server.paranoidLeakDetection ++ // Paranoid leak detection (affects performance)
-      Server.app(fooBar +++ app)      // Setup the Http app
+    Server.port(PORT) ++                                // Setup port
+      Server.paranoidLeakDetection ++                   // Paranoid leak detection (affects performance)
+      Server.app(HttpApp(fooBar.asHttp +++ app.asHttp)) // Setup the Http app
```

Comment:
```suggestion
      Server.app(fooBar +++ app) // Setup the Http app
```

- [ ] Add `+++` in `HttpApp`